Command/ImportMappingDoctrineCommand.php

```diff
@@ -59,9 +61,19 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $bundle = $this->getApplication()->getKernel()->getBundle($input->getArgument('bundle'));
+        if ($input->getOption('bundle-less')) {
```

Comment:
Much like with previous patches attempting to understand where files are located, I don't really understand why the user can't simply  perform the import and then copy the files over.

Reply:
Can you import if there is no single bundle in your app?

Reply:
There's surely at least DoctrineBundle, since you installed it? :P

Reply:
Obviously you are right, I didn't think about this kind of workaround - but still it's not perfect :)